### DISCUSSION TITLE
Could you add a feature for the top 500 english words

### DISCUSSION THREAD

**Discussion by one-loop** [monkeytypegame/monkeytype#921] [Ideas] [answered]
I was practicing typing on monkeytype and would usually get above 80wpm. However, in normal typing I would only type around 65wpm. I'm pretty sure this is because I only get 80+wpm for the top 200 words in english, but in normal typing, I'm typing much more than the most frequent 200 words. I know there is a feature to toggle the 1000 most common words, but that seems like too much. Can a feature for the 500 most common words be implemented.

If not, no worries.
Thanks

**Comment by Miodec** [comment, marked as answer]
I think this is a bit too specific, and I feel like there is already too many word lists. Sorry

**Comment by one-loop** [comment]
Yeah, no problem I understand now. 1000 words is fine, I've been practicing
on it for a while now.

On Fri, 12 Feb 2021, 7:03 am Jack, <notifications@github.com> wrote:

> I think this is a bit too specific, and I feel like there is already too
> many word lists. Sorry
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Miodec/monkeytype/discussions/921#discussioncomment-361503>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASTZRDTRN62N52CXF3ARRMLS6SAQDANCNFSM4XAOZL5Q>
> .
>

-- 
·         D I S C L A I M E R
The information transmitted is intended only 
for the person or organisation to which it is addressed and may contain 
confidential and/or privileged material. Any review, retransmission, 
dissemination or other use of, or taking of any action in reliance upon, 
this information by persons or organisation other than the intended 
recipient is prohibited. If you receive this in error, please contact the 
sender and delete the material from any computer.
This email and its 
contents have been scanned for viruses, but the scanner may not be able to 
detect or repair all viruses and variants. Please be aware that there is a 
risk involved whenever downloading email attachments to your computer or 
sending email attachments to others. Kuwait English School and its software 
providers accept no liability for loss or damage incurred as a result of 
downloading email attachments.
